01_Build_Process/05_Generating_Iso.md

```diff
@@ -49,7 +49,7 @@ The process for limine is a little more manual: we must build the iso ourselves,
 
 To get started we'll want to create a working directory to use as the root of our iso. In this case we'll use `disk/`. Next we'll need to clone the latest binary branch of limine (using `git clone https://github.com/limine-bootloader/limine.git --branch=v3.0-branch-binary --depth=1`) and copy 'limine.sys', 'limine-cd.bin', and 'limine-cd-efi.bin' into `disk/limine/`, creating that directory if it doesn't exist.
```

Comment:
lol this is showing the age of this part of the book: can you also update the branch name to `v8.x-binary` instead of `v3.0-branch-binary`?

Reply:
maybe we mention that this is the latest at the time of writing, but the reader should check the git repo themselves.